lib/ProofSet.js

```diff
@@ -77,18 +69,9 @@ module.exports = class ProofSet {
     }
 
     // preprocess document to prepare to remove existing proofs
-    let input;
-    if(compactProof) {
-      // cannot assume security context terms, so do full compaction
-      input = await jsonld.compact(
-        document, constants.SECURITY_CONTEXT_URL,
-        {documentLoader, expansionMap, compactToRelative: false});
-    } else {
-      // TODO: optimize to modify document in place to maximize optimization
-
-      // shallow copy document to allow removal of existing proofs
-      input = {...document};
-    }
+    // let input;
+    // shallow copy document to allow removal of existing proofs
+    const input = {...document};
 
     // save but exclude any existing proof(s)
     const proofProperty = 'proof';
```

Comment:
There's only one proof property now, (signature being removed), so we don't need this variable.

Reply:
Removed.